vectors/bnadd.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "Input": "089142debb13c461f61523586a60732d8b69c5b38a3380a74da7b2961d867dbf2d5fc7bbc013c16d7945f190b232eacc25da675c0eb093fe6b9f1b4b4e107b3625f8c89ea3437f44f8fc8b6bfbb6312074dc6f983809a5e809ff4e1d076dd5850b38c7ced6e4daef9c4347f370d6d8b58f4b1d8dc61a3c59d651a0644a2a27cf",
+    "Expected": "0a6678fd675aa4d8f0d03a1feb921a27f38ebdcb860cc083653519655acd6d79172fd5b3b2bfdd44e43bcec3eace9347608f9f0a16f1e184cb3f52e6f259cbeb",
+    "Name": "input_param_scalar_0_gas_150.csv-1"
+  },
+  {
+    "Input": "23f16f1bcc31bd002746da6fa3825209af9a356ccd99cf79604a430dd592bcd90a03caeda9c5aa40cdc9e4166e083492885dad36c72714e3697e34a4bc72ccaa21315394462f1a39f87462dbceb92718b220e4f80af516f727ad85380fadefbc2e4f40ea7bbe2d4d71f13c84fd2ae24a4a24d9638dd78349d0dee8435a67cca6",
+    "Expected": "013f227997b410cbd96b137a114f5b12d5a3a53d7482797bcd1f116ff30ff1931effebc79dee208d036553beae8ca71afb3b4c00979560db3991c7e67c49103c",
+    "Name": "input_param_scalar_0_gas_150.csv-1"
```

Comment:
Is it expected to name be always the same? Also, can I encode gas limit to JSON format (extend it with new field e.g. "MaxGas"?)

Reply:
The name was supposed to differ, so that's some error in my generation of it. I don't think gas makes sense to have in there -- the same vector could/should be used regardless of which gas formula is used